stubs/django/test/__init__.pyi

```diff
@@ -11,6 +11,5 @@ from django.test.client import Client as Client
 from django.test.utils import override_settings as override_settings
 
 class SimpleTestCase(TestCase):
-    client: testing.client.base.TestClient = ...
```

Comment:
To whomever reviews this, `testing` isn't imported by this stub. As best I can tell, `client` is set by `test_client`, which defaults to `django.test.client.Client` but can take any class that has the same API. Most likely this will be a subclass of `Client`, but if you think an interface is better here, let me know. (Perhaps that's what `TestClient` is intended to be.)